scripts/migrate.ts

```
@@ -0,0 +1,226 @@
+        break;
+      case '--database':
+        options.databaseName = args[++i];
+        break;
+      case '--schema':
+        options.schemaFile = args[++i];
+        break;
+      case '--help':
+        printHelp();
+        process.exit(0);
+      default:
+        console.error(`Unknown argument: ${args[i]}`);
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)